### ISSUE TITLE
ReferenceError: initialUrlValue is not defined at getParamValue 

### ISSUE THREAD

**Issue by arthabus** [buhrmi/vue-sync#9]
Out of a sudden I've started to get the below error

Here is the stacktrace I get:

```
ReferenceError: initialUrlValue is not defined at getParamValue 
(webpack://my_project/node_modules/vue-sync/index.js:82:24) at syncFn 
(webpack://my_project/node_modules/vue-sync/index.js:169:37) at map 
(webpack://my_project/node_modules/vue-sync/index.js:166:34) at call 
(webpack://my_project/node_modules/vue/dist/vue.esm.js:1872:56) at invokeWithErrorHandling 
(webpack://my_project/node_modules/vue/dist/vue.esm.js:4244:6) at callHook 
(webpack://my_project/node_modules/vue/dist/vue.esm.js:5031:4) at _init 
(webpack://my_project/node_modules/vue/dist/vue.esm.js:5177:11) at 

```

Checking the source code, initialUrlValue is not defined indeed:

line 82:      initialUrlValue = getParamValue(param); //should be   **let** initialUrlValue = getParamValue(param);

But I believe I was using it for quite a while and only today started to get this error.

version:
"vue-sync": "^3.2.0"

**Comment by buhrmi** [comment]
should be fixed in 3.2.1

**Comment by arthabus** [comment]
thanks, appreciate your lightning fast fix!
